c_emulator/riscv_sim.c

```diff
@@ -1088,15 +1093,29 @@ int main(int argc, char **argv)
     }
     printf("Connected\n");
   } else
-    entry = load_sail(file);
+    entry = load_sail(initial_elf_file);
 #else
-  uint64_t entry = load_sail(file);
+  uint64_t entry = load_sail(initial_elf_file);
 #endif
+  /* Load any additional ELF files into memory */
+  for (int i = files_start + 1; i < argc; i++) {
+    bool is32bit;
+    uint64_t additional_entry;
+    fprintf(stdout, "Loading additional ELF file %s.\n", argv[i]);
```

Comment:
Hm, we don't print the main file, seems a bit odd to do it for these

Reply:
We print the entry point for the main file. I can remove this, just thought it might be helpful.

Reply:
I'd missed that process_args prints it